act-rules-format.bs

```diff
@@ -245,183 +263,210 @@ For other aspects that are not well defined, an ACT Rule MUST include either a d
 
 <div class="example">
-  <p>A composite rule for <a href="https://www.w3.org/WAI/WCAG21/quickref/#audio-description-or-media-alternative-prerecorded" target="_blank">WCAG 2.1 Audio Description or Media Alternative</a> applies a logic across results from three atomic rules. The expectation of the composite rule is as follows:</p>
-  <blockquote>
-    <p>For each test target, the outcome of one of the following rules is `passed`:</p>
-    <ul>
-      <li>Video elements have an audio description (video-audio-desc)</li>
-      <li>Video elements have a media alternative (video-text-alt)</li>
-    </ul>
-  </blockquote>
+  <p>Composite rule: video elements have an audio description or media alternative ([WCAG 2.1, success criterion 1.2.3 Audio Description or Media Alternative](https://www.w3.org/WAI/WCAG21/quickref/#audio-description-or-media-alternative-prerecorded)).</p>
+  <p>Each HTML `video` element meets all expectations from at least one of the following rules:</p>
+  <ul>
+    <li>video elements have a transcript</li>
+    <li>video elements have an audio description</li>
+    <li>video elements have a description track</li>
+  </ul>
 </div>
 
 
 Assumptions {#assumptions}
-==========================
+--------------------------
 
-An ACT Rule MUST list any assumptions, limitations or any exceptions for the evaluation, the test environment, technologies being used or the subject being tested. For example, a rule that would partially test <a href="https://www.w3.org/WAI/WCAG20/quickref/#visual-audio-contrast-contrast" target="_blank">WCAG 2.0 Success Criterion 1.4.3 Contrast (Minimum)</a> based on the inspection of CSS properties could state that it is only applicable to HTML text content stylable with CSS, and that the rule does not support images of text.
+An ACT Rule MUST list any known assumptions, limitations or any exceptions for the evaluation, the test environment, technologies being used or the subject being tested. For example, a rule that would partially test [WCAG 2.1 Success Criterion 1.4.3 Contrast (Minimum)](https://www.w3.org/WAI/WCAG21/quickref/#visual-audio-contrast-contrast) based on the inspection of CSS properties could state that it is only applicable to HTML text content styleable with CSS, and that the rule does not support images of text.
 
 Sometimes there are multiple plausible ways that an accessibility requirement can be interpreted. For instance, it is not immediately obvious if emoji characters are "text" or "non-text content" under WCAG 2.1. Whatever the interpretation is, this MUST be documented in the rule.
 
 While this item MUST be included in the ACT Rule, it MAY be empty when there are no known assumptions, limitations or exceptions.
```

Comment:
**Editorial**: In most places in the spec, we talk about "parts of a rule", rather than "items". We should be consistent in this.

Reply:
"the assumptions"